src/mechanics/elasticity.cc

```diff
@@ -395,6 +399,12 @@ void Elasticity::initialize()
     eq_fields_->lame_lambda.set(Model<3, FieldValue<3>::Scalar>::create(fn_lame_lambda(), eq_fields_->young_modulus, eq_fields_->poisson_ratio), 0.0);
     eq_fields_->dirichlet_penalty.set(Model<3, FieldValue<3>::Scalar>::create(fn_dirichlet_penalty(), eq_fields_->lame_mu, eq_fields_->lame_lambda), 0.0);
 
+    // read optional user fields
+    Input::Array user_fields_arr;
+    if (input_rec.opt_val("user_fields", user_fields_arr)) {
+       	eq_fields_->init_user_fields(user_fields_arr, time_->step());
```

Comment:
How to deal with the time step?

Reply:
It seems that we have tests for reading the user fields in different equations. We should also test output of the user fields and test user fields of different shape. This common test would rather be in 03_output